### PR TITLE
Update model saving for new date saved key

### PR DESCRIPTION
**Pull request by fengwuyao**
Add new key for TFJS model corresponding to Keras V3 format.
- Update the saving model function when save the model
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

### CODE REVIEW

File: tfjs-layers/src/model_save_test.ts [hunk 1]
```diff
@@ -53,6 +53,14 @@ describeMathCPUAndGPU('LayersModel.save', () => {
     expect(handler.savedArtifacts.convertedBy).toEqual(null);
   });
 
+  it('Save model artifact with dateSaved key', async () => {
+    const model = tfl.sequential();
+    const handler = new IOHandlerForTest();
+    model.add(tfl.layers.dense({units: 1, inputShape: [3]}));
+    await model.save(handler);
+    expect(handler.savedArtifacts.dateSaved).not.toBeNull();
```
Comment (mattsoulanille) [inline tfjs-layers/src/model_save_test.ts:61]: Nit: Can we test for the value being date-like? This regex isn't perfectly specific, but it approximates what we expect.
```suggestion
    expect(handler.savedArtifacts.dateSaved).toMatch(/\d+-\d+\d+@\d+:\d+:\d+/);```

**Review by mattsoulanille — APPROVED** [review]
LGTM with a nit

**Comment by Linchenn** [comment]
Is this PR still valid? Otherwise, we could close it.

**Comment by fengwuyao** [comment]
Thanks for pointing out. Removed since not needed.

**Pull request closed without merging**